src/test/java/com/artipie/asto/RxFileTest.java

```diff
@@ -74,18 +74,20 @@ public void rxFileSaveWorks() throws IOException {
         final Vertx vertx = Vertx.vertx();
         final String hello = "hello-world!!!";
         final Path temp = Files.createTempFile(hello, "saved.txt");
-        temp.toFile().delete();
-        new RxFile(temp, vertx.fileSystem())
-            .save(
-                Flowable.fromArray(new ByteArray(hello.getBytes()).boxedBytes()).map(
-                    aByte -> {
-                        final byte[] bytes = new byte[1];
-                        bytes[0] = aByte;
-                        return ByteBuffer.wrap(bytes);
-                    }
-                )
-            ).blockingAwait();
-        MatcherAssert.assertThat(new String(Files.readAllBytes(temp)), Matchers.equalTo(hello));
+        for (int idx = 0; idx < 100; idx += 1) {
```

Comment:
@Sammers21 the build is failing now with the following error:
```
Failed tests: 
  RxFileTest.rxFileSaveWorks:89 
Expected: "hello-world!!!"
     but: was "helloworld!!!"
```

Link to build: https://github.com/artipie/asto/pull/50/checks?check_run_id=440602444